src/lib/api/filter_general.ts

```diff
@@ -0,0 +1,45 @@
+import { createClient } from "../client/supabase/server";
+
+export async function filter_general(
+  op: string,
+  column: string,
+  values: string[]
+) {
+  const valid_columns = [
+    "name_org",
+    "pseudonym",
+    "pronouns",
+    "email",
+    "phone",
+    "position",
+    "opt_in_communication",
+  ];
+
+  if (!valid_columns.includes(column)) {
+    return { data: null, error: "Invalid column name" };
+  }
+
+  const client = await createClient();
```

Comment:
This should be moved to be after all of the validation checks. This way we don't have to create a client before returning due to values being empty.